### PR TITLE
Get CSS styles programatically

### PR DESCRIPTION
**Pull request by Tranquility2**
As  @petargyurov advised, added the ability to fetch the CSS styles programmatically.
This should allow easier handling for new styles of books :)

### CODE REVIEW

**Comment by petargyurov** [comment]
Thanks @Tranquility2 ! Looks good, I only have some minor comments/questions:

- is the function `getAllCssRules` necessary provided it's only being used in `getRootCssStyles`?
- things might get messed up if we add other root variables that are used for something else; might be better to prefix all patterns vars... 🤔 ...I think it's fine for now, we can deal with that later if we come to it.
- I don't think we need to `console.log` anything 
- the file `css_utils.js` should probably be called `cssUtils.js`

**Comment by Tranquility2** [comment]
@petargyurov Thanks for the feedback!

- absolutely right, I usually keep the logic separated for future use, but I agree this will be simpler just to merge.
- good catch, I've updated all the patterns vars to avoid any future issues.
- commented out the console print (didn't deleted it as it can be useful for debugging issues), hope its ok. 
- totaly agree, update the file name accordingly. 

TL;DR added/changed everything as you recommended.
Please note I didn't squash all the commits (with the fixes) so it will be (hopefully) easier to review :) 
(If it looks ok **don't forget to squash merge**)

**Review by petargyurov — APPROVED** [review]

**Comment by Tranquility2** [comment]
Oh well, I assume you wanted the full history?😅

**Comment by petargyurov** [comment]
Nope, just trigger finger! Morning coffee hadn't kicked in yet 😅 Oh well.
